layers/+readers/pdf/packages.el

```diff
@@ -118,3 +119,9 @@
         "r"              'pdf-occur-revert-buffer-with-args
         "*"              'spacemacs/enter-ahs-forward
         "?"              'evil-search-backward))))
+
+(defun pdf/init-pdf-view-restore ()
+  (use-package pdf-view-restore
+    :after pdf-tools
+    :config
+    (add-hook 'pdf-view-mode-hook 'pdf-view-restore-mode)))
```

Comment:
Hooks should go into the init section to trigger lazy loading.

Reply:
Ai! I should have learned that by now... :grin: